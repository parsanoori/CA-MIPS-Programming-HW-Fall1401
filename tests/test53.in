72
485546616
463515253
136757174
801890863
580184706
203490826
841812581
397111989
1026595354
123028353
1010875275
320281475
515396332
713630232
659740774
307861119
941344484
419982560
895625438
921675330
160994802
350895228
700006667
290074997
469310246
1069203311
116908390
879890420
305890703
914379421
1042881514
180046689
70751807
60398839
525532031
1052517112
987635664
1051280275
591877266
823504855
441249831
346577147
670153779
487974945
583230434
979990320
764117162
758602768
716270608
559513221
508245921
407872284
978015297
246333712
488021764
81764812
268813408
73830587
42785642
880801374
106507613
1002334398
1035275259
693706372
211387974
143052520
384153088
1011073897
727711135
824558811
441463958
787352278