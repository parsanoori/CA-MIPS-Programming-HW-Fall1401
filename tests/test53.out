42785642
60398839
70751807
73830587
81764812
106507613
116908390
123028353
136757174
143052520
160994802
180046689
203490826
211387974
246333712
268813408
290074997
305890703
307861119
320281475
346577147
350895228
384153088
397111989
407872284
419982560
441249831
441463958
463515253
469310246
485546616
487974945
488021764
508245921
515396332
525532031
559513221
580184706
583230434
591877266
659740774
670153779
693706372
700006667
713630232
716270608
727711135
758602768
764117162
787352278
801890863
823504855
824558811
841812581
879890420
880801374
895625438
914379421
921675330
941344484
978015297
979990320
987635664
1002334398
1010875275
1011073897
1026595354
1035275259
1042881514
1051280275
1052517112
1069203311